_docs/templates/email-template.md

```diff
@@ -270,17 +356,19 @@ message type. It should match the content in the header of the email.
 
 {% endcolumn %}
 
-{% column right:"hxCol-md-8" %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
 {% figure [caption:"Mobile app promotion and download links"] [class:"image bg-light border"] %}
-  ![]({{site.cdn_url}}/img/layout/email-templates/Mobile%20LInks.svg){:width=60%}
+  ![]({{site.cdn_url}}/img/layout/email-templates/Mobile%20LInks.svg)
```

Comment:
s/`prodcut`/`product`
<img width="105" alt="screen shot 2017-11-20 at 11 17 07 am" src="https://user-images.githubusercontent.com/1538692/33031614-6c9a59d6-cde4-11e7-89e6-d260fc8acea4.png">

Reply:
I will need to rebuild this image.